cpp/cmake_modules/FindPythonLibsNew.cmake

```diff
@@ -246,7 +248,7 @@ FUNCTION(PYTHON_ADD_MODULE _NAME )
       SET_TARGET_PROPERTIES(${_NAME} PROPERTIES LINK_FLAGS
                           "-undefined dynamic_lookup")
     ELSEIF(MSVC)
-      target_link_libraries(${_NAME} ${PYTHON_LIBRARIES})
+      target_link_libraries(${_NAME} ${PYTHON_LIBRARIES} $<$<CONFIG:Debug>:${PYTHON_DEBUG_LIBRARIES}> $<$<CONFIG:Release>:${PYTHON_LIBRARIES}>)
```

Comment:
`${PYTHON_LIBRARIES}` is duplicated in the `Release` configuration.

Reply:
:+1: